src/gui/transferlistfilterswidget.cpp

```diff
@@ -488,7 +488,8 @@ void TrackerFiltersList::addItem(const QString &tracker, const QString &hash)
         trackerItem = new QListWidgetItem();
         trackerItem->setData(Qt::DecorationRole, GuiIconProvider::instance()->getIcon("network-server"));
 
-        downloadFavicon(QString("http://%1/favicon.ico").arg(host));
+        if(Preferences::instance()->isLoadFaviconsEnabled())
```

Comment:
You are locking a mutex for each tracker. Wouldn't it be better to cache option value?